packages/hoppscotch-common/src/components/http/TestResult.vue

```diff
@@ -197,11 +197,18 @@
       />
     </div>
     <EnvironmentsMyDetails
-      :show="showModalDetails"
+      :show="showMyEnvironmentDetailsModal"
       action="new"
       :env-vars="getAdditionVars"
       @hide-modal="displayModalAdd(false)"
     />
+    <EnvironmentsTeamsDetails
+      :show="showTeamEnvironmentDetailsModal"
+      action="new"
+      :env-vars="getAdditionVars"
+      :editing-team-id="workspace.type === 'team' ? workspace.teamID : null"
```

Comment:
there's tiny a type error here.


```suggestion
      :editing-team-id="workspace.type === 'team' ? workspace.teamID : undefined"
```